.github/workflows/release.yml

```diff
@@ -1,60 +1,111 @@
-#       - 'package-lock.json'
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - main
+    paths:
+      - 'src/**'
+      - 'test/**'
+      - 'scripts/**'
+      - 'public/**'
+      - 'packages/**'
+      - 'package.json'
+      - 'package-lock.json'
 
-# permissions:
-#   contents: read
+permissions:
+  contents: read
 
-# jobs:
-#   create_github_release:
-#     outputs:
-#       full-tag: ${{ steps.release-drafter.outputs.tag_name }}
-#       short-tag: ${{ steps.get_tag_name.outputs.SHORT_TAG }}
-#       body: ${{ steps.release-drafter.outputs.body }}
-#     runs-on: ubuntu-latest
-#     permissions:
-#       contents: write
-#       pull-requests: read
-#     steps:
-#       - uses: release-drafter/release-drafter@v6
-#         id: release-drafter
-#         env:
-#           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-#         with:
-#           publish: true
-#       - name: Get the short tag
-#         id: get_tag_name
-#         run: |
-#           short_tag=$(echo ${{ steps.release-drafter.outputs.tag_name }} | cut -d. -f1)
-#           echo "SHORT_TAG=$short_tag" >> $GITHUB_OUTPUT
-#   create_npm_release:
-#     needs: create_github_release
-#     runs-on: ubuntu-latest
-#     permissions:
-#       packages: write
-#     env:
-#       REGISTRY: ghcr.io
-#       IMAGE_NAME: ${{ github.repository }}
-#     steps:
-#       - uses: actions/checkout@8459bc0 # v4
-#       - uses: actions/setup-node@c2ac33f # v4, Setup .npmrc file to publish to npm
-#         with:
-#           node-version: '18.x'
-#           registry-url: 'https://registry.npmjs.org'
-#       - run: npm ci
-#       - run: npm publish --access=public
-#         env:
-#           NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
+jobs:
+  update_version:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+      - name: Run Release Drafter
+        uses: release-drafter/release-drafter@v6
+        id: release-drafter
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      - name: Extract Version
+        id: extract_version
+        run: |
+          version="${{ steps.release-drafter.outputs.tag_name }}"
+          version=$(echo "$version" || sed 's/^v//')
+          if [ -z "$version"]; then
+            echo "Version is empty. Exiting"
+            exit 1
+          fi
+          echo "version=$version" >> $GITHUB_OUTPUT
+      - name: Update package.json and package-lock.json version
+        run: 
+          VERSION=${{ steps.extract_version.outputs.version }}
+          if [ -z "$version"]; then
+            echo "Version is empty. Exiting"
+            exit 1
+          fi
+          jq --arg version "$VERSION" '.version = $version' package.json > package_tmp.json && mv package_tmp.json package.json
+          jq --arg version "$VERSION" '.version = $version' package-lock.json > package_lock_tmp.json && mv package_lock_tmp.json package-lock.json
```

Comment:
Instead of this step, can we update the `package.json` only and then run `npm install`? This will automatically update the `package-lock.json` for us.

Reply:
sure would make this change

Reply:
Awesome, thank you @divinetettey 👍